osmosis/assetlist.json

```diff
@@ -26512,6 +26512,58 @@
           }
         }
       ]
+    },
+    {
+      "description": "WLF coin",
+      "denom_units": [
+        {
+          "denom": "ibc/3D00ACF371FC6B7BC871399B1909DDE18749FA19DE6B7A4F74E1D96BC073B3BC",
+          "exponent": 0,
+          "aliases": [
+            "factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/wirelibertyfence"
+          ]
+        },
+        {
+          "denom": "wirelibertyfence",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/3D00ACF371FC6B7BC871399B1909DDE18749FA19DE6B7A4F74E1D96BC073B3BC",
+      "name": "wirelibertyfence",
+      "display": "wirelibertyfence.xyz",
+      "symbol": "WLF",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "chihuahua",
+            "base_denom": "factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/wirelibertyfence",
+            "channel_id": "channel-7"
+          },
+          "chain": {
+            "channel_id": "channel-113",
+            "path": "transfer/channel-113/factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/wirelibertyfence"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/wirelibertyfence.png"
```

Comment:
Please remove the duplicate image upload under /osmosis/images/, and update this URI to point to the chihuahua/images/ location.